control_server/main.go

```diff
@@ -3,26 +3,35 @@ package main
 import (
 	"flag"
 	"fmt"
-	"log"
+	"os"
 	"strconv"
 
 	"baas/pkg/api"
 
 	"baas/control_server/httpserver"
 	"baas/control_server/machines"
 	"baas/control_server/pixieserver"
+	log "github.com/sirupsen/logrus"
 )
 
 var (
 	static = flag.String("static", "control_server/static", "Static file dir to server under /static/")
 )
 
+func init() {
+	log.SetFormatter(&log.TextFormatter{ForceColors: true})
+	log.SetOutput(os.Stdout)
+	log.SetLevel(log.DebugLevel)
```

Comment:
Maybe we should make log level configurable possibly using an environment variable?